app/lib/three_scale/policies/specification.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module ThreeScale
+  module Policies
+    class Specification
+      extend ActiveModel::Naming
+      extend ActiveModel::Translation
+
+      def initialize(body)
+        @errors = ActiveModel::Errors.new(self)
+
+        begin
+          @doc = JSON.parse(body.to_s)
+        rescue JSON::ParserError
+          @errors.add(:base, :invalid_json)
+        end
+
+        @doc ||= {}
+      end
+
+      attr_reader :errors, :doc
+
+      def valid?
+        return false if errors.any?
+        JSON::Validator.fully_validate(JSON_SCHEMA, doc).each { |error| errors.add(:base, error) }
+        errors.empty?
+      end
+
+      def self.setup_json_validator!
+        JSON::Validator.schema_reader = JSON::Schema::Reader.new(accept_uri: false, accept_file: false)
```

Comment:
That will also set it for `ThreeScale::Swagger`

Reply:
😱 🤦‍♀️ 👍

Reply:
Probably fine if it is the same for both classes but it would be nice to have it in one single place then

Reply:
I think that it is not the same 😂 So I will address this 👍 😄